### PR TITLE
WIP: feat(Settings): added settings layout comp

### PR DESCRIPTION
**Pull request by TanyaDolgopolova**
Fixed design on general settings
Close #16 
Close #55

### CODE REVIEW

**Review by kvokov — APPROVED** [review]

**Comment by kvokov** [comment]
👍

File: src/components/Header.tsx [hunk 1]
```diff
@@ -35,14 +42,27 @@ const Header: React.FC<HeaderProps> = ({ children, links = [] }) => {
             to={link.to}
             key={i}
             activeStyle={{
-              color: '#1D1D1D',
```
Comment (cobraz) [inline src/components/Header.tsx:38]: Should this be added to `theme.colors` maybe? Maybe you can decide, @AnneMatilde?

File: src/modules/settings/components/LayoutSettings.tsx [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+            <></>
+          </OptionView>
+          <OptionView
+            completedText="Valgt"
+            contentName="Work Sans + Roboto"
+            loadingText="Bruk"
+            onPreviewClick={handleFontPreview}
+            onSelectClick={handleFontChoose}
+            previewText="Forhåndsvis"
+          >
```
Comment (cobraz) [inline src/modules/settings/components/LayoutSettings.tsx:146]: This should probably be a list that we can pull for the API. For now, we can add it as an array, but later @dem1k can add that to the backend, what do you say?

**Review by cobraz — APPROVED** [review]
Left a few comments, but this looks good to me :)

**Comment by cobraz** [comment]
@TanyaDolgopolova You can merge stuff on your own, when you've got the reviews you need :)

File: src/components/Header.tsx (hunk 1 above)
Comment (AnneMatilde) [reply src/components/Header.tsx:38]: Please replace the color #1D1D1D with #1B202E.

**Comment by AnneMatilde** [comment]
Hi @TanyaDolgopolova! Have you finished with the design implementation for this? 
image

**Comment by TanyaDolgopolova** [comment]
> Hi @TanyaDolgopolova! Have you finished with the design implementation for this?
> image

This is still in progress. I will add this list of views to the kit project and connect it to this component. For now I left it like that

File: src/modules/settings/components/LayoutSettings.tsx (hunk 2 above)
Comment (TanyaDolgopolova) [reply src/modules/settings/components/LayoutSettings.tsx:146]: I think the best option for this is create list component in Kit project, like we did with checkboxlist. I thinking about create an array which I will pass in parameters as a variant

**Comment by cobraz** [comment]
This is still WIP, correct? Can you add a `WIP:` syntax while it is?

**Comment by TanyaDolgopolova** [comment]
> This is still WIP, correct? Can you add a `WIP:` syntax while it is?

It still in progress. I will add it

**Comment by cobraz** [comment]
This has become stale, closing it. Reopen if need be. 👍

**Pull request closed without merging**